laws/src/main/scala/cats/laws/FoldableLaws.scala

```diff
@@ -8,6 +8,16 @@ import scala.collection.mutable
 trait FoldableLaws[F[_]] extends UnorderedFoldableLaws[F] {
   implicit def F: Foldable[F]
 
+  def foldRightLazy[A](fa: F[A]): Boolean = {
+    var i = 0
+    F.foldRight(fa, Eval.now("empty")) { (_, _) =>
+        i += 1
```

Comment:
We use += here but not below. Any reason for the difference?

Reply:
no real reason except this one is written by me and the other ones are from before. Just changed the other ones to be more consistent.